src/loading_zone.rs

```diff
@@ -4,7 +4,7 @@ use anyhow::anyhow;
 use serde::{Serialize, Serializer};
 
 #[derive(Clone, Eq, PartialEq, Hash, Debug)]
-pub struct LoadingZone(u8);
+pub struct LoadingZone(pub u8);
```

Comment:
This member must be public, because it disables us to do any verification on its value. Use `LoadingZone::try_from(42)` instead.